erizo_controller/erizoClient/src/views/VideoPlayer.js

```diff
@@ -1,179 +1,160 @@
-/*global window, document, L, webkitURL*/
-'use strict';
+/* global window, document, L, webkitURL, Erizo*/
+this.Erizo = this.Erizo || {};
+
 /*
  * VideoPlayer represents a Licode video component that shows either a local or a remote video.
  * Ex.: var player = VideoPlayer({id: id, stream: stream, elementID: elementID});
  * A VideoPlayer is also a View component.
  */
-var Erizo = Erizo || {};
-Erizo.VideoPlayer = function (spec) {
-    var that = Erizo.View({}),
-        onmouseover,
-        onmouseout;
-
-    // Variables
-
-    // VideoPlayer ID
-    that.id = spec.id;
-
-    // Stream that the VideoPlayer will play
-    that.stream = spec.stream.stream;
-
-    // DOM element in which the VideoPlayer will be appended
-    that.elementID = spec.elementID;
-
-    // Private functions
-    onmouseover = function () {
-        that.bar.display();
-    };
-
-    onmouseout = function () {
-        that.bar.hide();
-    };
-
-    // Public functions
-
-    // It will stop the VideoPlayer and remove it from the HTML
-    that.destroy = function () {
-        that.video.pause();
-        delete that.resizer;
-        that.parentNode.removeChild(that.div);
-    };
-
-    that.resize = function () {
-
-        var width = that.container.offsetWidth,
-            height = that.container.offsetHeight;
-
-        if (spec.stream.screen || spec.options.crop === false) {
-
-            if (width * (9 / 16) < height) {
-
-                that.video.style.width = width + 'px';
-                that.video.style.height = (9 / 16) * width + 'px';
-
-                that.video.style.top = -((9 / 16) * width / 2 - height / 2) + 'px';
-                that.video.style.left = '0px';
-
-            } else {
-
-                that.video.style.height = height + 'px';
-                that.video.style.width = (16 / 9) * height + 'px';
-
-                that.video.style.left = -((16 / 9) * height / 2 - width / 2) + 'px';
-                that.video.style.top = '0px';
-
-            }
-        } else {
-            if (width !== that.containerWidth || height !== that.containerHeight) {
-
-                if (width * (3 / 4) > height) {
-
-                    that.video.style.width = width + 'px';
-                    that.video.style.height = (3 / 4) * width + 'px';
-
-                    that.video.style.top = -((3 / 4) * width / 2 - height / 2) + 'px';
-                    that.video.style.left = '0px';
-
-                } else {
-
-                    that.video.style.height = height + 'px';
-                    that.video.style.width = (4 / 3) * height + 'px';
-
-                    that.video.style.left = -((4 / 3) * height / 2 - width / 2) + 'px';
-                    that.video.style.top = '0px';
-
-                }
-            }
-        }
-
-        that.containerWidth = width;
-        that.containerHeight = height;
+Erizo.VideoPlayer = (spec) => {
```

Comment:
Consider `Erizo.VideoPlayer = function VideoPlayer(spec)`
Arrow functions cannot be constructors